cli-deploy.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-require("./command-runner").run(async () => {
+
```

Comment:
Oh, good call

Reply:
Yeah, duh 🤦 It makes sense that this wouldn't need the command runner, since it's more of an entrypoint to register subcommands (just like `cli.js`). Didn't even think of it when I was going back and retrofitting all of the parent-level commands.

Reply:
Damnit @brandongregoryscott we demand perfection at all times 😄